.github/workflows/cmake.yml

```diff
@@ -107,14 +134,14 @@ jobs:
         ${{ toJSON(github.event) }}
         EOF
     - name: Generate IRC message
-      # I really dislike putting so much logic here, but I can't come up with something better.
-      if: github.repository == 'SerenityOS/serenity' && !cancelled() && (github.event_name == 'pull_request' || (github.event_name == 'push' && github.ref == 'refs/heads/master'))
+      if: matrix.should-notify-irc && !cancelled()
```

Comment:
This generated an IRC notification message when I pushed to my fork and I don't understand why. It was the same commit as this PR though..

https://github.com/ADKaster/serenity/runs/1672988057?check_suite_focus=true

Reply:
looks like I fixed this by checking `matrix.should-notify-irc == true` instead of `matrix.should-notify-irc`.